src/openforms/emails/digest.py

```diff
@@ -289,7 +297,7 @@ def collect_invalid_certificates() -> list[InvalidCertificate]:
             invalid_certs.append(
                 InvalidCertificate(
                     id=cert.id,
-                    label=cert.label,
+                    label=cert.__str__(),
```

Comment:
Firstly I changed this manually in the template (using a default filter for the empty label) but then I thought it's better since we have the `__str__` method doing the same thing. Let me know if you agree.